demos/src/js/demo.js

```diff
@@ -0,0 +1,114 @@
+var demo = demo || {};
+
+/*
+
+Potential demo color stacks:
+
+Grey Scale: colors: ["#979798", "#bbbdc0", "#58585b", "#e6e7e8", "#231f20"]
+Green: colors: ["#00a59a", "#9bc863", "#1a6b61", "#e9ea7b", "#1b443d"]
+Blue: colors: ["#04ca6dc", "#65cdf5", "#546ab2", "#b6e2ec", "#20386c"]
+Red: colors: ["#f15e4e", "#faa634", "#a41c3f", "#fde95f", "#5b173b"]
+Varied#1: colors: ["#04858d", "#41beae", "#faa634", "#f15e4e", "#004861"]
+Varied#2: colors: ["#f7961c", "#87acdb", "#ffc60a", "#a2ce5a", "#f15e4e"]
+Varied#3: colors: ["#f15e4e", "#acdee4", "#73c163", "#ffc74a", "#41beae"]
+
+*/
+
+(function ($, fluid) {
+
+    "use strict";
+
+    fluid.registerNamespace("demo.chartAuthoring");
+
+    demo.chartAuthoring.addExampleInput = function (that) {
+        var dataEntries = that.chartAuthoringInterface.dataEntryPanel.locate("dataEntries");
+
+        var labelInputs = dataEntries.find("input.floec-ca-dataEntry-label");
```

Comment:
Just noticed, ```label``` and ```value``` selectors belong to ```dataEntry``` components that are sub and dynamic components of dataEntryPanel. That's probably why you didn't use locate to find them.

Since all dynamic components are generated in order with names "dataEntry", "dataEntry-1", "dataEntry-2" and so on, we can loop thru and use locate() to find label and value and set initial values.